uvdb.c

```diff
@@ -308,7 +309,7 @@ static void write_x(size_t sz)
         buffer_write(&out_buf, "xx", 2);
 }
 
-static size_t safe_memcpy(char* dst, const char* src, size_t sz)
+static size_t safe_sceClibMemcpy(char* dst, const char* src, size_t sz)
```

Comment:
Looks like you did a find-replace on the whole source code. But this one does not make any sense.

Reply:
oops. fixed